repository/v.json

```diff
@@ -483,6 +483,17 @@
 					"branch": "master"
 				}
 			]
+		},
+		{
+			"name": "VvPHPDollar",
+			"details": "https://github.com/ZhaonanLi/VvPhpDollar/releases/tag/1.0.0",
```

Comment:
Hi FichteFoll,

I have modified the code and the v.json file.
Thanks a lot!

Sincerely,
Zhaonan Li.